javascript/javascript3/week1/giphy exrercise/giphy.js

```diff
@@ -0,0 +1,38 @@
+/////// Giphy APP
+
+const gifButton = document.getElementById("btn");
+const yourSearh = document.getElementById("gifSearch");
+const showGif = document.getElementById("number");
```

Comment:
may be numbeOfGifs instead of showGif?
showGif sounds more like a boolean variable.